test/SecTester.Bus.Tests/Internal/JsonHttpMethodEnumerationStringConverterTests.cs

```diff
@@ -0,0 +1,59 @@
+namespace SecTester.Bus.Tests.Internal;
+
+public class JsonHttpMethodEnumerationStringConverterTests
```

Comment:
The test suite name and SUT are different, move all of them to `DefaultMessageSerializerTests`